webextension/privileged/ui/CloudDownloadsView.jsm

```diff
@@ -0,0 +1,743 @@
+      return;
+    }
+
+    // Create a duplicate download which will show the correct
+    // target path for moved session download in Download UI panel
+    const publicList = await Downloads.getList(Downloads.ALL);
+
+    const movedDownload = await Downloads.createDownload({
+      source: download.source,
+      target: destPath
+    });
+    movedDownload.startTime = download.startTime;
+    movedDownload.succeeded = true;
+    await publicList.add(movedDownload);
+
+    // Update destination path used in download history library panel
+    PlacesUtils.annotations.setPageAnnotation(
+      NetUtil.newURI(download.source.url),
+      "downloads/destinationFileURI",
+      "file://" + destPath, 0,
+      PlacesUtils.annotations.EXPIRE_WITH_HISTORY);
+
+    // Explicitly updates the state of a moved download
+    movedDownload.refresh().catch(Cu.reportError);
+
+    // Remove original download in favor of moved download from download UI panel
+    await publicList.remove(download);
+  },
+
+  async handleMove(moveDownloadMenuItem) {
+    if (!this.selectedProvider) {
+      return;
+    }
+
+    // Compute provider download folder path from selectedProvider object
+    const providerDownloadFolder = OS.Path.join(this.selectedProvider.value.downloadPath,
+      this.selectedProvider.value.typeSpecificData.default);
+
+    // create download directory if it doesn't exist
+    try {
+      await OS.File.makeDir(providerDownloadFolder, {ignoreExisting: true});
+    } catch (err) {
+      Cu.reportError(err);
+      return;
+    }
+
+    // Check if there is a download item to be moved to provider folder, if not exit
+    if (!moveDownloadMenuItem) {
+      return;
+    }
+
+    const document = CloudDownloadsView.browserWindow.document;
+    const downloadItemElements = document.getElementById("downloadsListBox").childNodes;
+    const download = downloadItemElements[moveDownloadMenuItem.getAttribute("itemIndex")]._shell.download;
+    if (download.succeeded) {
+      await this._moveDownload(download, providerDownloadFolder);
+    }
+  },
+
+  async handleLocalMove(moveDownloadMenuItem) {
+    const downloadsDir = await Downloads.getPreferredDownloadsDirectory();
+    const downloadsDirExists = await this.checkIfAssetExists(downloadsDir);
+    if (downloadsDirExists) {
+      const document = CloudDownloadsView.browserWindow.document;
+      const downloadItemElements = document.getElementById("downloadsListBox").childNodes;
+      const download = downloadItemElements[moveDownloadMenuItem.getAttribute("itemIndex")]._shell.download;
+      if (download.succeeded) {
+        await this._moveDownload(download, downloadsDir);
+      }
+    }
+  },
+};
+
+
+var WindowListener = {
+  setupBrowserUI: function wm_setupBrowserUI(window) {
+    const utils = window.QueryInterface(Ci.nsIInterfaceRequestor).getInterface(Ci.nsIDOMWindowUtils);
+    utils.loadSheetUsingURIString(CloudDownloadsView.stylesURL, Ci.nsIDOMWindowUtils.AGENT_SHEET);
```

Comment:
The reason you're needing to use `!important` is because you're registering as a user agent sheet. Register as an `AUTHOR_SHEET` and you shouldn't have this problem if your selectors are more specific than the ones in use by the browser.

Reply:
it works with AUTHOR_SHEET, thanks!